examples/advanced-project-js/src/__checks__/website-group.check.js

```diff
@@ -24,6 +24,11 @@ const websiteGroup = new CheckGroup('website-check-group-1', {
   apiCheckDefaults: {},
   concurrency: 100,
   alertChannels,
+  /*
+   * Failed check runs in this group will be retried before triggering alerts.
+   * The wait time between retries will increase linearly, with a wait of 30 seconds, 60 seconds, then 90 seconds between the retries.
+   */
+  retryStrategy: RetryStrategyBuilder.linearStrategy({ baseBackoffSeconds: 30, maxAttempts: 3, sameRegion: false }),
 });
 
-module.exports = websiteGroup;
+module.exports = { websiteGroup };
```

Comment:
There was a bug here since other `*.check.js` files were importing using:

```
const { websiteGroup } = require('./website-group.check');
```

This was causing the group to be empty when the advanced JS project is deployed.